### PR TITLE
Portfolio: Add Contact form

### PR DESCRIPTION
**Pull request by blessing-michael**
# Portfolio: add a contact form and finish the mobile version
 In this project I;
- Created a contact form
- Collect data by using the [Formspree service](https://formspree.io/html).
- I  used Flexbox to lay out the element on the page

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -340,6 +340,90 @@ body {
...
+  display: flex;
+  align-items: center;
+  border-top-left-radius: 5rem;
+  box-shadow: 2px 4px 4px rgba(0, 0, 0, 0.1);
+  flex-direction: column;
+  padding: 7rem 2rem 2rem 2rem;
+  font-family: poppins, sans-serif;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
```
Comment (AbelG101) [inline style.css]: - [ ] Ouch this is close but you seem to be missing 1 icon from the background image of the contact section as shown below in the original design 👇 please make sure to pay attention to small details like this as this is the trait of a great frontend developer.
- Figma design:
  image
- Your design:
  image

File: .vscode/settings.json
Comment (AbelG101) [inline .vscode/settings.json:1]: **THIS COMMENT IS REGARDING YOUR README AND NOT THIS FILE**
- [ ] I noticed the MIT link in your readme is broken so please fix that as it makes your project look unprofessional 👶

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @blessing-michael 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Form data is submitted properly  ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by joskalenda — APPROVED** [review]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @blessing-michael , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ... Great design
✅ ... Green linters

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**